### ISSUE TITLE
Undefined method 'default_session' for Mongoid:Module

### ISSUE THREAD

**Issue by MGalv** [adacosta/mongoid_rails_migrations#37]
Hey, great work! I've been using this gem recently an I've found it helpful, thanks!

I've come to get this issue when using the last released version of the gem, but it seems it has been solved on the master branch. Are you planing on realeasing a new version any time soon? That would be great.

Thanks again!

**Comment by Loschcode** [comment]
This issue does not seem to be fixed for me

**Comment by adacosta** [comment]
@Loschcode If you or anyone else has time to create a PR with the fix, I'd be happy to merge it. Thanks for leaving a comment on the issue status.

**Comment by cgriego** [comment]
As @MGalv pointed out, this issue is already fixed in master. Without this fix being released, there is no released version compatible with Mongoid 5+. It'd be really nice to get a new release made. @adacosta

**Comment by jarthod** [comment]
Master has been released (compatible with Mongoid 5+) as `1.1.1`

**Comment by jgonzalezd** [comment]
Posting here 'cos I just hit this error running a migration in Rails.
If you are seeing this error and you're calling a method inside 
```
def self.up
 common_method()
end

prvate
def common_method()
end
```
then you probably forgot to put the `self` on the common method too. `def self.common_method`.
